test/unit/three_scale/open_api/url_resolver_test.rb

```diff
@@ -39,6 +39,17 @@ class ThreeScale::OpenApi::UrlResolverTest < ActiveSupport::TestCase
     assert_equal ['https://api.example.com'], resolver.servers
   end
 
+  test 'url empty variable' do
+    specification = {
+      'servers' => [{
+        'url' => '{protocol}://api.example.com',
+        'variables' => {}
+      }]
+    }
+    resolver = UrlResolver.new(specification)
+    assert_equal ['https://api.example.com'], resolver.servers
```

Comment:
This should be a there is no `protocol` variable.

```suggestion
    assert_equal [''{protocol}://api.example.com'], resolver.servers
```

Reply:
fixed